src/core/ProductTile.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import Icon from "./Icon";
+import FeaturedLink from "./FeaturedLink";
+import { ProductName, products } from "./ProductTile/data";
+
+type ProductTileProps = {
+  name: ProductName;
+  selected?: boolean;
+  currentPage?: boolean;
+};
+
+const ProductTile = ({ name, selected, currentPage }: ProductTileProps) => {
+  const { icon, label, description, link, unavailable } = products[name] ?? {};
+
+  return (
+    <div
+      className={`rounded-lg p-12 flex flex-col gap-8 transition-colors ${selected ? "bg-neutral-300" : "bg-neutral-1200 hover:bg-neutral-1100"}`}
+    >
+      <div className="flex gap-12">
+        {icon && <Icon name={icon} size="48" />}
```

Comment:
wondering if `icon` is undefined should we use ternary Operator instead?

Reply:
Yep, you're right